tools/dialignr/dialignr.xml

```diff
@@ -0,0 +1,155 @@
+            <param name="analyteFDR" type="float" min="0" max="1" value="0.01" label="Analyte FDR" help="Defines the upper limit of FDR on a precursor to be considered for multipeptide." />
+            <param name="unalignedFDR" type="float" min="0" max="1" value="0.01" label="Unaligned FDR" help="Must be between 0 and maxFdrQuery. Features below unalignedFDR are considered for quantification even without the RT alignment." />
+            <!-- max="maxFdrQuery" -->
+            <param name="alignedFDR" type="float" min="0" max="1" value="0.05" label="Aligned FDR" help="Must be between unalignedFDR and 1. Features below alignedFDRare considered for quantification after the alignment." />
+            <!-- min="unalignedFDR" -->
+            <param name="baselineType" type="select" label="Baseline method" help="Method to estimate the background of a peak contained in XICs.">
+                <option value="base_to_base" selected="true">base to base</option>
+                <option value="vertical_division_min">vertical division min</option>
+                <option value="vertical_division_max">vertical division max</option>
+            </param>
+            <param name="integrationType" type="select" label="Integration method" help="Method to compute the area of a peak contained in XICs.">
+                <option value="intensity_sum" selected="true">intensity sum</option>
+                <option value="trapezoid">trapezoid</option>
+                <option value="simpson">simpson</option>
+            </param>
+            <param name="fitEMG" type="boolean" truevalue="TRUE" falsevalue="FALSE" label="Fit EMG" help="Enable/disable exponentially modified gaussian peak model fitting." />
+            <param name="recalIntensity" type="boolean" truevalue="TRUE" falsevalue="FALSE" label="Recal intensity" help="Recalculate intensity for all analytes." />
+            <param name="fillMissing" type="boolean" checked="true" truevalue="TRUE" falsevalue="FALSE" label="Fill missing" help="Calculate intensity for analytes for which features are not found." />
+            <param name="smoothPeakArea" type="boolean" truevalue="TRUE" falsevalue="FALSE" label="Smooth peak area" help="No: Raw chromatograms will be used for quantification. Yes: Smoothed chromatograms will be used for quantification." />
+        </section>
+    </inputs>
+    <outputs>
+        <data name="out_aligned_targeted_runs" format="tsv" label="${tool.name} on ${on_string}: alignedTargetedRuns.tsv" from_work_dir="alignedTargetedRuns.tsv" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input_mzml_files" ftype="mzml" value="hroest_K120809_Strep0_PlasmaBiolRepl2_R04_SW_filt,hroest_K120809_Strep10_PlasmaBiolRepl2_R04_SW_filt" />
+            <param name="input_osw_file" ftype="osw" value="merged.osw" />
+            <output name="out_aligned_targeted_runs" file="alignedTargetedRuns.tsv" />
```

Comment:
Nico could we add one test that uses some advanced parameters? The rest is awesome! Thanks.